templates/base/store/global/reducer.js

```diff
@@ -0,0 +1,22 @@
+import { APP_ERROR } from './actions';
```

Comment:
This is wrong, you need to use the same constant that used in action 
UPDATE_APP_ERROR

Reply:
fixed

Reply:
fixed.